### PR TITLE
Add clean state example

### PR DESCRIPTION
**Pull request by itegulov**
Hopefully this is what we are looking for but lmk if you need anything more in-depth

### CODE REVIEW

File: examples/clean-state/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Clean state contract in JavaScript
+
+This is an equivalent JavaScript implementation of the clean state example. Every user can send and receive a non-fungible token.
```
Comment (ailisp) [inline examples/clean-state/README.md:3]:
```suggestion
This is an equivalent JavaScript implementation of the clean state example. 
```

File: examples/clean-state/__tests__/test-clean-state.ava.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    const root = worker.rootAccount;
+
+    // Deploy the jsvm contract.
+    const jsvm = await root.createAndDeploy(
+        root.getSubAccount('jsvm').accountId,
+        './node_modules/near-sdk-js/res/jsvm.wasm',
+    );
+
+    // Deploy clean state contract
+    const cleanStateContract = await root.createSubAccount('fungible-token');
```
Comment (ailisp) [inline examples/clean-state/__tests__/test-clean-state.ava.js:24]:
```suggestion
    const cleanStateContract = await root.createSubAccount('clean-state');
```

File: examples/clean-state/src/index.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { NearContract, NearBindgen, call, view, near, LookupMap } from 'near-sdk-js'
+
+@NearBindgen
+class CleanState extends NearContract {
+    constructor(keys) {
+        super()
+        keys.forEach(key => near.jsvmStorageRemove(key))
+    }
```
Comment (ailisp) [inline examples/clean-state/src/index.js:8]: I suggest to make a separate method `cleanState`, would be more clear than reuse the constructor

File: examples/clean-state/README.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
```
Comment (ailisp) [inline examples/clean-state/README.md:4]: I suggest to add a short instruction about how to use this contract. In summary, the steps are:
1. Deploy this contract to the account where you want to clean up state
2. Call the contract with known state keys to clean state. If delete all keys doesn't fit gas limit, you may need to do it a few times
3. Once finished clean up state, Deploy the original contract or delete the contract code with `near js remove`

**Review by ailisp — APPROVED** [review]
Overall the implementation and test is good! Docs need some update to help new developers

File: examples/clean-state/README.md (hunk 1 above)
Comment (itegulov) [reply examples/clean-state/README.md]: Yeah I keep forgetting to update all the stuff, thanks for keeping your eyes open, haha

File: examples/clean-state/src/index.js (hunk 3 above)
Comment (itegulov) [reply examples/clean-state/src/index.js:8]: Makes sense, I just followed the Rust example which used an `#[init]` method

File: examples/clean-state/README.md (hunk 4 above)
Comment (itegulov) [reply examples/clean-state/README.md:4]: Done, PTAL

**Review by volovyk-s — APPROVED** [review]
Cool!
From now on we will need to add each example to the examples CI job (check `.github/` folder).

**Comment by itegulov** [comment]
@volovyk-s done!

**Comment by itegulov** [comment]
@volovyk-s seems like the CI has run out of space. Can we upgrade the instance?

**Review by ailisp — APPROVED** [review]
Looks great!

**Pull request merged**